media/css/firefox/android/all.less

```diff
@@ -0,0 +1,99 @@
+@import '../../sandstone/lib.less';
+
+#main-feature {
+    .span-all();
+    h1 {
+        margin-bottom: @baseLine;
+    }
+    h2 {
+        .font-size(24px);
+        letter-spacing: -0.25px;
+        line-height: 1.4;
+    }
+}
+
+#main-content {
+    .build-table-container {
+        .span-all();
+    }
+    table {
+        width: 100%;
+        thead {
+            th {
+                white-space: nowrap;
+                &:not([colspan]) {
+                    width: 15%;
+                }
+            }
+            th, td {
+                padding: 10px 0;
+            }
+        }
+        tbody {
+            th, td {
+                padding: 10px 0;
+            }
+        }
+        .download {
+            padding: 10px 0;
+            a {
+                display: block;
+                padding: 5px 5px 5px 35px;
+                background: transparent url(/media/img/firefox/android/android-download.png) left center no-repeat;
+                background: transparent url(/media/img/firefox/android/android-download.svg) left center no-repeat;
```

Comment:
Might some browsers download the SVG here and still not support it? I think there's a trick here to use https://css-tricks.com/a-complete-guide-to-svg-fallbacks/